lib/rouge/cli.rb

```diff
@@ -62,18 +65,22 @@ def initialize(message, status=1)
     def self.parse(argv=ARGV)
       argv = normalize_syntax(argv)
 
-      mode = argv.shift
+      while (head = argv.shift)
```

Comment:
Is the reason to wrap the `case` in a `while` so that a user can require multiple libraries? Should that be more obvious from the usage instructions?

Reply:
Yes, but also this is a standard way of parsing command line arguments, so that certain parts can use `argv.shift` in the body, which is impossible with a standard `each { ... }` loop.

Reply:
Gotcha.